src/store/redis/redis_nodeset.c

```diff
@@ -1052,6 +1052,9 @@ static void nodeset_cluster_check_event_callback(redisAsyncContext *ac, void *re
   
   for(i=0; i<n; i++) {
     l = &lines[i];
+    if(l->maybe_failed) {
+      nodeset_log_warning(ns, "fail? flag on node %s.", l->id.data);
```

Comment:
The log warning should be a little more descriptive about what action is being taken. In this case, something like:
```
nodeset_log_warning(ns, "Ignoring 'fail?' state reported by %s for node %s", node_nickname_cstr(node), l->id.data)
```